### DISCUSSION TITLE
how to build and deploy it under my own apple dev ID

### DISCUSSION THREAD

**Discussion by Shivaanonym** [creepymonster/GlucoseDirect#573] [General]
Hi, I am not very technical but I need to find a way to have the CGM value from libre 3 in the loop app, and I hope this app can help me doing that. I do not know how to build and deploy GlucoseDirect app in apple developer, under my own apple developer ID as it seems it should be so to be able to use it with the loop app or maybe I am totally wrong!!!

**Comment by gui-dos** [comment]
Similarly to Loop, iAPS and xDrip4iO5, just replace in https://github.com/creepymonster/GlucoseDirect/blob/main/GlucoseDirect.xcconfig the `##TEAM_ID##` placeholder with your effective Apple Team Id. You can also duplicate the `.xcconfig` file and rename it `GlucoseDirectOverride.xcconfig` so that your edits are kept outside of git commits.

**Reply by Shivaanonym** [reply]
Thank you
